### ISSUE TITLE
Implement all message types

### ISSUE THREAD

**Issue by Glodenox** [Glodenox/twitch-chat-monitor#5]
At the moment only, the message type "message" is shown. The other message types like bits, subscriptions, moderator actions, people entering or leaving, raids and basically any other event listed in [the tmi.js documentation](https://github.com/tmijs/docs/blob/gh-pages/_posts/v1.4.2/2019-03-03-Events.md).

Pretty much requires the settings window window to be available with issue #1 as otherwise there would be way too many messages in a somewhat active chat.

Minimum needed:
- [x] Cheer
- [x] Raided
- [x] Subscription
- [x] Resub
- [x] Subgift
- [x] Submysterygift
- [x] Messagedeleted
- [x] Clearchat

Nice to have:
- [x] Timeout
- [x] Ban
- [x] Hosting
- [x] Unhost

**Comment by Glodenox** [comment]
Removed hosted event as that requires the user to be logged in and removed the join and part events as these trigger only once every minute or so to provide an update of who is online - rendering it useless as a notification. It might still be useful to keep track of the amount of users in the chat maybe, but I suspect that's not a very useful metric for streamers.
